core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreaker.java

```diff
@@ -71,6 +71,21 @@ static CircuitBreaker ofDefaultName() {
 
     /**
      * Decides whether a request should be sent or failed depending on the current circuit state.
+     *
+     * @deprecated Use {@link #tryRequest()}.
      */
+    @Deprecated
     boolean canRequest();
+
+    /**
+     * Decides whether a request should be sent or failed depending on the current circuit state.
+     * If the current state is {@link CircuitState#OPEN} and {@link CircuitBreakerConfig#circuitOpenWindow()}
```

Comment:
Because `CircuitBreakerConfig` is not a public class, let's use `CircuitBreakerBuilder`.